index.md

```diff
@@ -259,268 +301,300 @@ Er is nu één vijand en deze staat recht tegenover de speler. Een stilstaande v
-	-- laad het plaatje van de vijand
-	vijandPlaatje = love.graphics.newImage('plaatjes/vijandsVliegtuig.png')
+-- laad het plaatje van de vijand
+vijandPlaatje = love.graphics.newImage('plaatjes/vijandsVliegtuig.png')
 {{< /highlight >}}
 			
 De vijanden moeten ook getekend worden, zorg ervoor dat jouw `love.draw(dt)` functie eruit ziet zoals hieronder:
 
 {{< highlight lua >}}
-	function love.draw(dt)
-	
-	  -- teken het plaatje op het scherm
-	  love.graphics.draw(speler.plaatje, speler.x, speler.y)
-	
-	  -- teken de vijanden in de lijst
-	  tekenVijanden(vijanden)
-	
-	end
+function love.draw(dt)
+
+  -- teken het plaatje op het scherm
+  love.graphics.draw(speler.plaatje, speler.x, speler.y)
+
+  -- teken de vijanden in de lijst
+  tekenVijanden(vijanden)
+
+end
 {{< /highlight >}}
 			
 Er worden alleen vijanden getekend die bestaan, daarom moeten vijanden ook aangemaakt worden:
 
 {{< highlight lua >}}
-	function love.update(dt)
-	
-	  maakNieuweVijand(vijanden)
+function love.update(dt)
+
+  maakNieuweVijand(vijanden)
 {{< /highlight >}}
 			
 In de vorige stap hebben we de vijand laten bewegen. Deze code moeten we nu voor iedere vijand in de lijst uitvoeren:
 
 {{< highlight lua >}}
-	function love.update(dt)
-	
-	  maakNieuweVijand(vijanden) 
-	
-	  -- voor elke vijand in de lijst  
-	  for index, vijand in ipairs(vijanden) do
-	
-	    -- laat de vijand een stapje naar beneden doen  
-	    vijand.y = vijand.y + 2
-	
-	    -- als de vijand de onderrand heeft bereikt
-	    if vijand.y > yOnderRand() then
-	
-	      -- verwijder de vijand
-	      table.remove(vijanden, index)
-	
-	    end
-	  end
+function love.update(dt)
+
+  maakNieuweVijand(vijanden) 
+
+  -- voor elke vijand in de lijst  
+  for index, vijand in ipairs(vijanden) do
+
+	-- laat de vijand een stapje naar beneden doen  
+	vijand.y = vijand.y + 2
+
+	-- als de vijand de onderrand heeft bereikt
+	if vijand.y > yOnderRand() then
+
+	  -- verwijder de vijand
+	  table.remove(vijanden, index)
+
+	end
+  end
 {{< /highlight >}}
 			
 Het maken en verwijderen van vijanden zoals we dat hierboven doen is in Scratch heel lastig. We geven daarom geen 
 voorbeeld van het bovenstaande in Scratch. Waar het kan doen we dat wel.
 
-Voer het programma uit. Als het goed is ziet het eruit als onderstaande plaatje:
+Voer het programma uit. Als het goed is ziet het eruit als in onderstaande plaatje:
 
 ![scherm met vijandelijke vliegtuigen](images/image3.png)
 
 ### Stap 6: de vijand ontwijken
 Als de speler een vijand raakt, is het spel afgelopen. Om te zien of de speler een vijand geraakt heeft, hebben we 
 functie `spelerHeeftVijandGeraakt()`.
 
-Als het resultaat van de functie waar (true) is, dan moet het spel gestopt worden.
+Als het resultaat van de functie waar (`true`) is, dan moet het spel gestopt worden.
 
 Voeg daarvoor de volgende code toe aan functie `love.update(dt)`:
 
-![Scratch spel is afgelopen](images/image2.png)
+<!--![Scratch spel is afgelopen](images/image2.png)-->
```

Comment:
Ik zie dat het origineel ook in deze volgorde was, maar ik zou liever eerst de Löve code en dan de Scratch blokken zien.

Reply:
Helemaal mee eens.